### DISCUSSION TITLE
Issue / PR titles are missing Explore feed

### DISCUSSION THREAD

**Discussion by ruudk** [community/community#145634] [Mobile] [answered]
Since the beginning I notice that there are no titles for issues or PR's mentioned in the Explore feed.

Why is that? It doesn't make any sense to hide that info. Often, it's _the_ trigger to know if you are interested in it or not.

Instead, we see the branch name and the body.

Please add the title.
![image](https://github.com/user-attachments/assets/79b55552-4606-439c-b43d-c66183a8c4c9)

On web, the titles _are_ shown:
![Screenshot 2024-11-26 at 11 19 14@2x](https://github.com/user-attachments/assets/e98d1452-f2b7-48e7-948f-e30a7015f4c5)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.186.0 (146974874)
Device: iPhone 15 Pro Max (iOS 18.1.1)
</details>

**Comment by iskae** [comment]
Hi @ruudk 👋 

I _believe_ you are using our iOS client, right? I have checked the Android, and we are showing the pull request titles in there. Reaching out to our iOS colleagues and creating an internal issue for it 🙇 

Thanks so much for reaching out and please stay tuned for our future releases ❤️

**Reply by ruudk** [reply]
Yes, I'm using GitHub for iOS, Version 1.186.0 (146974874) Device: iPhone 15 Pro Max (iOS 18.1.1)

**Reply by iskae** [reply, marked as answer]
Thanks! We were able to reproduce it! We will try to fix it as soon as possible 🙇

**Comment by thiner** [comment]
this is also occurring on desktop browser now. My explore page shows nothing but some recommended repos by GitHub.

**Comment by Pricer187** [comment]
> Since the beginning I notice that there are no titles for issues or PR's mentioned in the Explore feed.
> 
> 
> 
> Why is that? It doesn't make any sense to hide that info. Often, it's _the_ trigger to know if you are interested in it or not.
> 
> 
> 
> Instead, we see the branch name and the body.
> 
> 
> 
> Please add the title.
> 
> ![image](https://github.com/user-attachments/assets/79b55552-4606-439c-b43d-c66183a8c4c9)
> 
> 
> 
> On web, the titles _are_ shown:
> 
> ![Screenshot 2024-11-26 at 11 19 14@2x](https://github.com/user-attachments/assets/e98d1452-f2b7-48e7-948f-e30a7015f4c5)
> 
> 
> 
> 
> 
> <details>
> 
> <summary>ℹ️ Additional Debug Info</summary>
> 
> GitHub for iOS, Version 1.186.0 (146974874)
> 
> Device: iPhone 15 Pro Max (iOS 18.1.1)
> 
> </details>

195

**Comment by Pricer187** [comment]
> Since the beginning I notice that there are no titles for issues or PR's mentioned in the Explore feed.
> 
> 
> 
> Why is that? It doesn't make any sense to hide that info. Often, it's _the_ trigger to know if you are interested in it or not.
> 
> 
> 
> Instead, we see the branch name and the body.
> 
> 
> 
> Please add the title.
> 
> ![image](https://github.com/user-attachments/assets/79b55552-4606-439c-b43d-c66183a8c4c9)
> 
> 
> 
> On web, the titles _are_ shown:
> 
> ![Screenshot 2024-11-26 at 11 19 14@2x](https://github.com/user-attachments/assets/e98d1452-f2b7-48e7-948f-e30a7015f4c5)
> 
> 
> 
> 
> 
> <details>
> 
> <summary>ℹ️ Additional Debug Info</summary>
> 
> GitHub for iOS, Version 1.186.0 (146974874)
> 
> Device: iPhone 15 Pro Max (iOS 18.1.1)
> 
> </details>

195
